vrs/vrs-annotation-batch-wrapper.py

```diff
@@ -0,0 +1,245 @@
+usage: python3 vrs-annotation-batch-wrapper.py \
+    --billing-project gnomad-vrs \
+    --working-bucket gnomad-vrs \
+    --image us-central1-docker.pkg.dev/broad-mpg-gnomad/ga4gh-vrs/marten-vrs-image-v2-031323 \
+    --version test_v3_1k \
+    --prefix marten_prelim_test \
+    --partitions-for-vcf-export 20 \
+    --header-path gs://gnomad-vrs/header-fix.txt
+"""
+
+import argparse
+import datetime
+import errno
+import logging
+import os
+import sys
+
+import hail as hl
+import hailtop.batch as hb
+from gnomad.resources.grch38.gnomad import public_release
+from gnomad_qc.v3.resources.annotations import vrs_annotations as v3_vrs_annotations
+from tgg.batch.batch_utils import init_job
+
+logging.basicConfig(
+    format="%(asctime)s (%(name)s %(lineno)s): %(message)s",
+    datefmt="%m/%d/%Y %I:%M:%S %p",
+)
+logger = logging.getLogger("annotate_vrs_ids")
+logger.setLevel(logging.INFO)
+
+
+def init_job_with_gcloud(
+    batch,
+    name: str = None,
+    image: str = None,
+    cpu: float = None,
+    memory: float = None,
+    disk_size: float = None,
+    mount: str = None,
+):
+    """
+    Create job and initialize glcoud authentication and gsutil commands.
+    Wraps Ben Weisburd's init_job (https://github.com/broadinstitute/tgg_methods/blob/master/tgg/batch/batch_utils.py#L160) with additional gcloud steps.
+    Parameters passed through to init_job:
+        :param batch: Batch object.
+        :param name: Job label which will show up in the Batch web UI.
+        :param image: Docker image name (eg. "us-central1-docker.pkg.dev/broad-mpg-gnomad/ga4gh-vrs/marten-vrs-image-v2-031323").
+        :param cpu: Number of CPUs (between 0.25 to 16).
+        :param memory: Amount of RAM in Gb (eg. 3.75).
+        :param disk_size: Amount of disk in Gb (eg. 50).
+        :param mount: Name of GCP Bucket to mount using cloudfuse.
+        :return: New job object.
+    """
+    job = init_job(batch, name, image, cpu, memory, disk_size)
+    job.command(f"gcloud -q auth activate-service-account --key-file=/gsa-key/key.json")
+    job.command(f"curl -sSL broad.io/install-gcs-connector | python3")
+    job.cloudfuse(mount, "/local-vrs-mount")
+    return job
+
+
+def main(args):
+
+    # Working bucket definition
+    working_bucket = args.working_bucket 
+
+    # prefix to create custom named versions of outputs 
+    prefix = args.prefix + args.version 
+
+    # Input paths (fixed) - note that the Hail Tables are altered outside of the script and may partition oddly
+    input_paths_dict = {
+        "3.1.2": public_release("genomes").path ,
+        "test_v3_1k": "gs://gnomad-vrs/ht-inputs/ht-1k-TESTING-ONLY.ht",
+        "test_v3_10k": "gs://gnomad-vrs/ht-inputs/ht-10k-TESTING-ONLY.ht",
+        "test_v3_100k": "gs://gnomad-vrs/ht-inputs/ht-100k-TESTING-ONLY.ht",
+    }
+
+    output_paths_dict = {
+        "3.1.2": v3_vrs_annotations.path,
```

Comment:
can you test and confirm that you can write to this output running this script?

Reply:
Good that you brought this up - I've been having problems that I actually CANNOT write to the specific path supplied, since it's a Requester Pays Bucket and both KC and I have been having problems writing to such buckets in Query On Batch. There's an open Zulip thread about it in the #gnomad-ga4gh Slack Channel , or check it out here: https://hail.zulipchat.com/#narrow/stream/123010-Hail-Query-0.2E2-support/topic/Writing.20to.20requester.20pays.20bucket